index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Module 1 </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <div class="brown">
+        <section><ul><li>Portoflio</li>
+            <li>About</li>
+            <li>Conduct</li></ul>
+        </section>
+        <header class="header"> 
+            <h1>'Hey there. I'm Brown.'</h1>
+            <h2>I'm a software developer</h2>
+        </header>
+        <div class="support">
+<p>I can help you build a project, features or website looking throught some of
+    my works and experience! If you like what you see and have a project you need coded,
+    don't hestiate to contact me.
+</p></div>
+
+<div class="social">
+   <div class="twit"><ion-icon name="logo-twitter"></ion-icon></div>
+   <div class="git"><ion-icon name="logo-github"></ion-icon></div>
+    <div class="link"><ion-icon name="logo-linkedin"></ion-icon></div>
+    <div class=""><ion-icon name="logo-medium"></ion-icon></div>
+</div>
```

Comment:
- [ ] Kindly make this section an unordered list instead of divs. By doing this, you will have made your markup semantically correct.
- [ ] Also, make the icon images inside a link so that when someone clicks on it when you attach the link to be redirected to, they are directed there.

For a visual of what I mean, I will write for you a general level example:

```
<ul>
  <li><a href="#"><img src="yourimg" alt=""></a></li>
</ul>
```